src/js/structs/Job.js

```diff
@@ -102,7 +102,13 @@ module.exports = class Job extends Item {
   }
 
   getSchedules() {
-    return this.get('schedules') || [];
+    const schedules = this.get('schedules');
+
+    if (!Array.isArray(schedules)) {
+      return [];
```

Comment:
I'd rather test if it's an Object and wrap it with an array.

Because here we just discard user input without any message/error
so a user is able to create a job that will be unscheduled. Everybody knows how it is frustrating when a service just throws your input away without telling you anything. Let's not be that service 💪